Dockerfile

```diff
@@ -1,71 +1,50 @@
-# Build compilation image
-FROM ruby:3.1.4-alpine as builder
+FROM ruby:3.1.4-alpine3.18 as base
 
-# The application runs from /app
 WORKDIR /app
 
-# Add the timezone as it's not configured by default in Alpine
+# Add the timezone as it’s not configured by default in Alpine
 RUN apk add --update --no-cache tzdata && \
   cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
   echo "Europe/London" > /etc/timezone
 
-# build-base: complication tools for bundle
+# build-base: compilation tools for bundle
 # yarn: node package manager
 # postgresql-dev: postgres driver and libraries
-RUN apk add --no-cache build-base yarn postgresql-dev git
+RUN apk add --no-cache build-base yarn postgresql-dev git bash
 
-# Install bundler to run bundle exec
-# This should be the same version as the Gemfile.lock
+# Bundler version should be the same version as what the Gemfile.lock was bundled with
 RUN gem install bundler:2.3.14 --no-document
```

Comment:
Why is this specific version of `bundler` being used? Is it just what the devs were using at the time the `Gemfile.lock` file was originally created?

Reply:
Only a year old, so not super important, but you may as well update this to a newer version if you can.

Reply:
We'll mention this to Rachael / CORE Application team, and leaving as is for now